package.json

```diff
@@ -31,25 +31,25 @@
   "author": "Matteo Collina <hello@matteocollina.com>",
   "contributors": [
     {
-      "name": "Tomas Della Vedova",
-      "url": "http://delved.org"
-    },
```

Comment:
Tomas was completely removed.

Reply:
Not so. It is second in the list. Although, I have learned that the name should probably be alphabetized by "Vedova" instead of "Della".

Reply:
Nope, my surname is "Della Vedova" :P